src/SourceIndex/index.proj

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="15.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
+  <ItemGroup>
+    <SourceProject Include="../**/src/*.csproj"/>
+    <SourceProject Include="../**/tests/*.csproj"/>
+    <SourceProject Include="../**/src/*.vbproj"/>
+    <SourceProject Include="../**/tests/*.vbproj"/>
+    <SourceProject Include="../../../coreclr/src/mscorlib/System.Private.CoreLib.csproj"/>
```

Comment:
We should probably do some sort of exists check on this as most people won't have this.